recipes/libftdi/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class LibFtdiConan(ConanFile):
+    name = "libftdi"
+    description = "A library to talk to FTDI chips"
+    license = ["MIT", "BSD-3-Clause"]
+    topics = ("conan", "libftdi")
+    homepage = "http://developer.intra2net.com/git/?p=libftdi;a=summary"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake_find_package"
```

Comment:
The `cmake` generator is required to make sure the compiler is configured correctly.
```suggestion
    generators = "cmake", "cmake_find_package"
```

Reply:
#### Done

`cmake` added to generators list.